packages/react-core/src/components/ExpandableSection/ExpandableSectionToggle.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+import styles from '@patternfly/react-styles/css/components/ExpandableSection/expandable-section';
+import { css } from '@patternfly/react-styles';
+import AngleRightIcon from '@patternfly/react-icons/dist/js/icons/angle-right-icon';
+
+export interface ExpandableSectionToggleProps extends React.HTMLProps<HTMLDivElement> {
+  /** Content rendered inside the expandable toggle. */
+  children?: React.ReactNode;
+  /** Additional classes added to the expandable toggle. */
+  className?: string;
+  /** Flag indicating if the expandable section is expanded. */
+  isExpanded?: boolean;
+  /** Callback function to toggle the expandable content. */
+  onToggle?: (isExpanded: boolean) => void;
+  /** ID of the toggle's respective expandable section content. */
+  contentId?: string;
+}
+
+export const ExpandableSectionToggle: React.FunctionComponent<ExpandableSectionToggleProps> = ({
+  children,
+  className = '',
+  isExpanded = false,
+  onToggle,
+  contentId,
+  ...props
+}: ExpandableSectionToggleProps) => (
+  <div
+    {...props}
+    className={css(
+      styles.expandableSection,
+      isExpanded && styles.modifiers.expanded,
+      styles.modifiers.detached,
+      className
+    )}
+  >
+    <button
+      className={css(styles.expandableSectionToggle)}
+      type="button"
+      aria-expanded={isExpanded}
+      aria-controls={contentId}
+      onClick={() => onToggle(!isExpanded)}
+    >
+      <span className={css(styles.expandableSectionToggleIcon, isExpanded && styles.modifiers.expandTop)}>
```

Comment:
`styles.modifiers.expandTop` should probably be added via a prop that the user sets to indicate that the expanded content is above the toggle. By default, it's fair to assume the content is below the toggle (just like in the default use of the component), so the toggle arrow should point down when expanded.